wow/sql/alerts_lagged_eviction_filings.sql

```diff
@@ -1,9 +1,10 @@
 SELECT
-	COUNT(DISTINCT INDEXNUMBERID) AS LAGGED_EVICTION_FILINGS
+  COUNT(DISTINCT INDEXNUMBERID) AS LAGGED_EVICTION_FILINGS,
+  MAX(I.FILEDDATE) AS LAGGED_EVICTION_DATE
 FROM OCA_METADATA AS M
 INNER JOIN OCA_INDEX AS I USING(INDEXNUMBERID)
 INNER JOIN OCA_ADDRESSES_WITH_BBL AS A USING(INDEXNUMBERID)
 WHERE A.BBL = %(bbl)s
   AND I.CLASSIFICATION = ANY('{Holdover,Non-Payment}')
   AND I.PROPERTYTYPE = 'Residential'
-  AND M.INITIALDATE > %(prev_date)s and I.FILEDDATE < %(prev_date)s
+  AND M.INITIALDATE > %(prev_date)s and I.FILEDDATE < %(prev_date)s and I.FILEDDATE > %(oldest_filed_date)s
```

Comment:
FYI my original approach for capping the eviction filings to be no older than 90 days was to do an inline calculation here (see code snippet below) to avoid adding an unnecessary API param. this didn't work so i ended up adding the query param after all. if you think there's a better approach, i'm all ears! 👂🏻
```
I.FILEDDATE > %(prev_date - INTERVAL '90 days')s
```

Reply:
Amazing thanks so much for jumping in and doing so much to fix this! I think it's fine now with the extra param, but I think your original idea would work with a slight adjustment. The `%()s` is just for the python string interpolation, so  I think it needs to just be the `prev_date` variable, and then the rest of the SQL can stay outside. 
```I.FILEDDATE > %(prev_date)s  - INTERVAL '90 days'```

Reply:
oooo i see. this would've been much simpler 🤦🏻‍♀️
good to know for if/when we want to get rid of this param